tests/Zend/Filter/DecryptTest.php

```diff
@@ -36,13 +36,30 @@
  */
 class Zend_Filter_DecryptTest extends PHPUnit_Framework_TestCase
 {
+    protected $errorReporting;
+
     public function setUp()
     {
+        // mcrypt is deprecated in PHP 7.1 (but still installed by default on Travis)
+        // hiding deprecated errors so tests pass.
+        if (substr(PHP_VERSION, 0, 3) === '7.1') {
```

Comment:
This is the same code that I used in #7 just moved to each test class's setup method.